code/collections.rb

```diff
@@ -1,15 +1,18 @@
 # collections
 
 # 1. Create a variable `point1` and assign it a hash representing a coordinate with keys `:x` and `:y` which have integer values
-
+point1={
+    x: 25,
+    y: 32
+}
 # 2. Print the y value
-
+point1[:y]
 # 3. Make an array, `points`, that includes `point1` and two more points (`points.length` should be 3)
-
+points= [point1, point2, point3]  assuming point2= {x:27, y:30} point3= {x:31, y:35}
```

Comment:
I understand your intention, but this isn't valid ruby code. Your variable assignments needed to be put onto separate lines and in the right order.

```ruby
point2= {x:27, y:30}
point3= {x:31, y:35}
points= [point1, point2, point3]
```

alternatively, you could have created the new hashes inside of the array, without assigning them to variables first:
```ruby
points= [point1, {x:27, y:30}, {x:31, y:35}]
```